setup.py

```diff
@@ -14,7 +14,6 @@
 # limitations under the License.
 
 from setuptools import setup
-from setuptools.command.test import test as TestCommand
```

Comment:
Should we also remove the `tests_require`-related stuff?

Reply:
We've discussed this here: https://github.com/IBM/python-sdk-core/pull/203#pullrequestreview-2205389145